pkgs/development/tools/build-managers/buildbot/default.nix

```diff
@@ -61,22 +57,26 @@ pythonPackages.buildPythonApplication (rec {
   preInstall = ''
     # writes out a file that can't be read properly
     sed -i.bak -e '69,84d' buildbot/test/unit/test_www_config.py
+
+    # un-hardcode path to tail
+    sed -i.bak 's|/usr/bin/tail|tail|' buildbot/scripts/logwatcher.py
   '';
 
   postFixup = ''
-    buildPythonPath "$out"
```

Comment:
this pr is updating the existing wrapper with the correct shebang.

im not sure why the `PYTHONPATH` is not being set properly by `buildPythonApplication`, but without this wrapper, buildbot does not start up. it just hangs with no output.  this is related to https://github.com/NixOS/nixpkgs/pull/18667